test/sampletests/withverify/verifyWithinPerform.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  before(client) {
+    client.globals.calls++;
+  },
```

Comment:
`before` and `after` hooks are unnecessary over here. Do you think we should remove them?

Reply:
removed